test/fixtures/transaction_builder.json

```diff
@@ -632,7 +632,7 @@
       },
       {
         "description": "Incomplete transaction w/ prevTxScript defined",
-        "exception": "Transaction is missing signatures",
+        "exception": "Not enough signatures provided",
```

Comment:
> Transaction is missing signatures

To be honest,  this is actually an error case where something has gone wrong internally.

`.signatures` is now always defined if the number of signatures required is known.
Therefore,  its a matter of "not enough" or "too many signatures" instead.